texar/modules/encoders/multihead_attention.py

```diff
@@ -255,6 +255,6 @@ def _combine_heads(self, x: torch.Tensor) -> torch.Tensor:
 
     @property
     def output_size(self):
-        r"""Provides output dimension as property.
+        r"""Return the finale dimension of :meth:`forward` output tensor.
```

Comment:
Should be "final" but not "finale". "Finale" means the last piece of music in a performance.